app/controllers/applications_controller.rb

```diff
@@ -1,9 +1,29 @@
 class ApplicationsController < ApplicationController
 
+  def show 
+    @application = Application.find(params[:id])
+  end
+  
   def new
+    @new_application = Application.new
   end
 
   def create
+    @new_application = Application.create(app_params)
+
+    if @new_application.save
```

Comment:
Will this create a redundant save call? If I remember correctly .create implicitly calls .save. Could this produce redundant entries in the database? I am unsure.